[2022-12-11T16:31:09.566+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: hard_dag.file_exist scheduled__2022-12-11T16:30:00+00:00 [queued]>
[2022-12-11T16:31:09.579+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: hard_dag.file_exist scheduled__2022-12-11T16:30:00+00:00 [queued]>
[2022-12-11T16:31:09.579+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-11T16:31:09.580+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-11T16:31:09.580+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-11T16:31:09.595+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): file_exist> on 2022-12-11 16:30:00+00:00
[2022-12-11T16:31:09.599+0000] {standard_task_runner.py:55} INFO - Started process 26771 to run task
[2022-12-11T16:31:09.602+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'hard_dag', 'file_exist', 'scheduled__2022-12-11T16:30:00+00:00', '--job-id', '7613', '--raw', '--subdir', 'DAGS_FOLDER/hard_dag.py', '--cfg-path', '/tmp/tmps2g8ich_']
[2022-12-11T16:31:09.603+0000] {standard_task_runner.py:83} INFO - Job 7613: Subtask file_exist
[2022-12-11T16:31:09.665+0000] {task_command.py:376} INFO - Running <TaskInstance: hard_dag.file_exist scheduled__2022-12-11T16:30:00+00:00 [running]> on host 5663d391ad54
[2022-12-11T16:31:09.739+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hard_dag
AIRFLOW_CTX_TASK_ID=file_exist
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T16:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-11T16:30:00+00:00
[2022-12-11T16:31:09.755+0000] {base.py:71} INFO - Using connection ID 'postgre_con' for task execution.
[2022-12-11T16:31:09.787+0000] {sql.py:315} INFO - Running statement: CREATE TABLE if not exists rnd_table (id serial PRIMARY KEY, value_1 integer, value_2 integer), parameters: None
[2022-12-11T16:31:09.803+0000] {base.py:71} INFO - Using connection ID 'postgre_con' for task execution.
[2022-12-11T16:31:09.839+0000] {sql.py:315} INFO - Running statement: truncate rnd_table, parameters: None
[2022-12-11T16:31:09.895+0000] {logging_mixin.py:137} INFO - INSERT INTO rnd_table (value_1,value_2) VALUES (9, 7)
[2022-12-11T16:31:09.904+0000] {base.py:71} INFO - Using connection ID 'postgre_con' for task execution.
[2022-12-11T16:31:09.944+0000] {sql.py:315} INFO - Running statement: INSERT INTO rnd_table (value_1,value_2) VALUES (9, 7), parameters: None
[2022-12-11T16:31:09.958+0000] {sql.py:324} INFO - Rows affected: 1
[2022-12-11T16:31:09.966+0000] {logging_mixin.py:137} INFO - INSERT INTO rnd_table (value_1,value_2) VALUES (9, 6)
[2022-12-11T16:31:09.973+0000] {base.py:71} INFO - Using connection ID 'postgre_con' for task execution.
[2022-12-11T16:31:10.002+0000] {sql.py:315} INFO - Running statement: INSERT INTO rnd_table (value_1,value_2) VALUES (9, 6), parameters: None
[2022-12-11T16:31:10.008+0000] {sql.py:324} INFO - Rows affected: 1
[2022-12-11T16:31:10.013+0000] {logging_mixin.py:137} INFO - INSERT INTO rnd_table (value_1,value_2) VALUES (8, 2)
[2022-12-11T16:31:10.024+0000] {base.py:71} INFO - Using connection ID 'postgre_con' for task execution.
[2022-12-11T16:31:10.074+0000] {sql.py:315} INFO - Running statement: INSERT INTO rnd_table (value_1,value_2) VALUES (8, 2), parameters: None
[2022-12-11T16:31:10.080+0000] {sql.py:324} INFO - Rows affected: 1
[2022-12-11T16:31:10.084+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-11T16:31:10.095+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=hard_dag, task_id=file_exist, execution_date=20221211T163000, start_date=20221211T163109, end_date=20221211T163110
[2022-12-11T16:31:10.152+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-11T16:31:10.175+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
